commands/application.react.js

```diff
@@ -124,6 +125,17 @@ const install = (options, appPath, styles) => {
     runCommand('npm', ['install'], { cwd: appPath });
 };
 
+const addPolyfills = (appPath) => {
```

Comment:
Better pass `indexFilePath` and `packagePath` as arguments (instead of `appPath`). The `packagePath` is already defined on the [line 102](https://github.com/DevExpress/devextreme-cli/pull/74/files#diff-eaaef3f02bea64f7eeee0dc7eb315298R100).
```suggestion
const addPolyfills = (packagePath, indexFilePath) => {
```

Reply:
Refactored